common/djangoapps/course_modes/tests/test_models.py

```diff
@@ -31,6 +31,9 @@ def setUp(self):
         self.course_key = SlashSeparatedCourseKey('Test', 'TestCourse', 'TestCourseRun')
         CourseMode.objects.all().delete()
 
+    def tearDown(self):
+        invalidate_course_mode_cache(sender=None)
```

Comment:
Ah, I see - it's not always called as a signal handler. Carry on then, disregard the above.